services/galley/galley.integration.yaml

```
@@ -30,6 +34,7 @@ settings:
   concurrentDeletionEvents: 1024
   deleteConvThrottleMillis: 0
   enableIndexedBillingTeamMembers: true
+  # enableFederationWithDomain: wire.com
```

so this is redundant in both brig and galley?  *sigh* brig and galley need to be merged.

```suggestion
  # enableFederationWithDomain: wire.com
  # NOTE: enableFederationWithDomain needs to have the same value as the corrresponding setting in brig.
```

(and perhaps a similar comment in brig.)

---

added (https://github.com/wireapp/wire-server/pull/1133/commits/6f9d03f052da82305c0e3f06c1ee40ad39c82485)